src/core/frames/frame_controller.ts

```diff
@@ -427,17 +427,43 @@ export class FrameController
   }
 
   private handleFrameMissingFromResponse(fetchResponse: FetchResponse) {
-    this.showFrameMissingError()
-    this.throwFrameMissingException(fetchResponse)
+    if (this.responsePermittedToBreakOutOfFrame(fetchResponse)) {
+      this.warnFrameMissingBreakout(fetchResponse)
+      this.visitResponse(fetchResponse.response)
+    } else {
+      this.showFrameMissingError()
+      this.throwFrameMissingException(fetchResponse)
+    }
+  }
+
+  private responsePermittedToBreakOutOfFrame(fetchResponse: FetchResponse) {
+    const allowedPathsTag = this.element.ownerDocument.querySelector<HTMLMetaElement>(
+      `meta[name=turbo-frame-breakout-paths]`
```

Comment:
Could do with a better name for this tag, I think.

Reply:
I think `turbo-frame-escape-paths` would be appropriate. Escape has a similar meaning to "breakout", but it's more suggestive of this being configuration for an exceptional case: an escape hatch.

Reply:
Yes! That's much better 👍